third-party/realdds/src/control.cpp

```diff
@@ -15,7 +15,7 @@ char dummy;
 }  // namespace
 #endif  // _WIN32
 
-#include "control.h"
```

Comment:
Can we find a way to avoid this change for auto generated files?
We wish to keep it as minimal, currently it's only copyrights

Reply:
The only two ways are to keep the files together or add the topic include directories (all of them) to the include directory list.

I don't see doing either. We'll have to manually change every time the .idl changes, but that shouldn't happen once we finalize...

Reply:
@OhadMeir  What do you think about it?, any suggestions?

Reply:
I suggest putting all the topic files together under realdds/topics.
Don't separate the generated h and cpp files

Reply:
You're suggesting that the include directory would contain .cpp files, which is what we have today.
You don't want the .cpp file together.
They only do this in the generated code because they have no way of knowing how things sit in the tree, They don't even know to convert `device-info` to `deviceInfo`, and they put their own license on things. Changes are inevitable.

Reply:
not under the `include` directory. on a third directory besides the `include` and `src`

Reply:
So you're saying add another include directory, that has both .h and .cpp in it?

Reply:
It doesn't have to be an include directory in the project definitions. You can use <realdds/topics/...> to include files from it in dds_device_watcher etc.

Reply:
I don't understand at all.
Outside of realdds, others need access to these headers.
So the .h files need to be in a public include directory.
Call me if we need to talk.

Reply:
As we discussed, I am OK with accepting this change